new/LIT7084QeneNeus.xml

```diff
@@ -0,0 +1,87 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7084QeneNeus" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ንኡስ፡ አነ፡ እምአሀውየ፡ እመኒ፡ እለብስ፡ መአርገ፡ እሉ፡ ሎግዮ፤</title>
```

Comment:
I would end the line with ፡, I would skip the explanation "(breastplate of the high priest)" in the title.